lib/system/osalloc.nim

```diff
@@ -87,6 +87,8 @@ elif defined(posix):
     const MAP_ANONYMOUS = 0x1000
   elif defined(solaris):
     const MAP_ANONYMOUS = 0x100
+  elif defined(linux):
+    const MAP_ANONYMOUS = 0x20
```

Comment:
@Araq it can happens to BSD, but i dont think kernel dependent constant will be different for Linux distros.